### DISCUSSION TITLE
Suggenstion for Commit messages

### DISCUSSION THREAD

**Discussion by tictaqqn** [community/community#13110] [Copilot Conversations]
Hi! I enjoy using Copilot!
I want Copilot to generate commit messages if possible.
Thank you!

**Comment by ynigun** [comment]
You can add Copilot to vim
https://www.reddit.com/r/ProgrammerHumor/comments/onezeb/github_copilot_helping_me_write_commit_messages/
or edit the commit messages in VScode
https://stackoverflow.com/questions/30149132/multiline-git-commit-message-in-vscode

**Reply by langestefan** [reply]
I hate vim with a passion. Does copilot even read your commited changes as text when you setup a commit message?

**Comment by nomike** [comment]
Automatically generating commit comments sounds like it could be a stronghold for copilot. It would need to be implemented though I guess.
